flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1852,6 +1852,42 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "EXTRACT(DOY FROM f16)",
       "315")
 
+    testSqlApi(
+      "EXTRACT(EPOCH FROM f18)",
+      "847608883")
```

Comment:
Why does the PostgreSQL (which seems to be the only major DB that supports EPOCHs?) returns different results here? The results are 10 seconds of, which seems disturbing.

http://sqlfiddle.com/#!17/9eecb/21451

It might have something to do with leap seconds...